src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -1107,7 +1107,11 @@ class RecipeMarkdownGenerator : Runnable {
                 newLine()
                 writeln("## Contributors")
                 for (contributors in recipeDescriptor.contributors) {
-                    writeln("* [${contributors.name}](${contributors.email})")
+                    if (contributors.email.contains("noreply")) {
+                        writeln("* ${contributors.name}")
+                    } else {
+                        writeln("* [${contributors.name}](mailto:${contributors.email})")
+                    }
```

Comment:
Alternatively we can just always use the name only, or not create mail to links. Up to reviewer.
```suggestion
                    writeln("* ${contributors.name}")
```

Reply:
Confirmed fixed in https://openrewrite.github.io/rewrite-recipe-markdown-generator/reference/recipes/java/spring/security5/upgradespringsecurity_5_8.html
![image](https://github.com/openrewrite/rewrite-recipe-markdown-generator/assets/1027334/81049303-190a-4130-b331-e84c4d10c222)